### PR TITLE
Add reload method to search again when there is request_id and payment_id

### PR DESCRIPTION
**Pull request by antoniofilho**

### CODE REVIEW

**Comment by j133y** [comment]
:rocket:

File: lib/braspag-rest/sale.rb [hunk 1]
```diff
@@ -54,6 +54,14 @@ def capture(amount = nil)
       payment.captured?
     end
 
+    def reload
+      if !request_id.nil? && payment && !payment.id.nil?
```
Comment (bvicenzo) [inline lib/braspag-rest/sale.rb]: Could this condition be like this?
```request_id.present? && payment.try(:id).present?```

**Comment by bvicenzo** [comment]
:shipit:

File: lib/braspag-rest/sale.rb (hunk 1 above)
Comment (antoniofilho) [inline lib/braspag-rest/sale.rb]: I don't have active support dependency here...

Comment (bvicenzo) [inline lib/braspag-rest/sale.rb]: :+1:

**Comment by no-donut-for-you** [comment]
:facepunch:

**Pull request merged**